src/app/logger.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoggerService {
```

Comment:
is this going to be modified to do real logging eventually?

Reply:
Yes this should change.